eslint.config.js

```
@@ -0,0 +1,22 @@
+      globals: {
+        ...globals.browser,
+      },
+    },
+  },
+  {
+    ignores: [
+      "packages/backend/vitest.config.ts",
+      "packages/frontend/vitest.config.ts",
+    ],
+  },
+];
```

ESLint

[error] Insert `⏎`

(prettier/prettier)